config/v1/0000_00_cluster-version-operator_01_clusterversion.crd.yaml

```diff
@@ -70,6 +70,7 @@ spec:
                           - Insights
                           - Storage
                           - CSISnapshot
+                          - node-tuning
```

Comment:
NodeTuning is going to be the preferred syntax/casing.

openshift-samples/baremetal/marketplace are grandfathered in.